xrootd/xrdproto/signing.go

```diff
@@ -96,16 +97,20 @@ func NewSignRequirements(level SecurityLevel, overrides []SecurityOverride) Sign
 
 	if level >= Compatible {
 		// TODO: set requirements
+		sr.requirements[open.RequestID] = SignLikely
```

Comment:
is the `TODO` still needed?

Reply:
Well, we need to add requirements for the most of requests, so I think we can leave `TODO` here to be sure.

Reply:
ok.